phoenix-protocol/src/main/PTable.proto

```diff
@@ -57,6 +57,8 @@ message PTableStats {
   optional int64 keyBytesCount = 4;
   optional int32 guidePostsCount = 5;
   optional PGuidePosts pGuidePosts = 6;
+  optional bytes encodedGuidePosts = 7;
```

Comment:
I'm a bit confused by this, though. Doesn't the PTable still send multiple PGuidePosts (one per column family)? It's just the PGuidePosts that have changed, no? You already have maxLength on PGuidePosts (which is where it belongs). Why is it needed here?